examples/matlab/robot_ocp/log_robot.txt

```diff
@@ -0,0 +1,5 @@
+---------------------------------------
```

Comment:
No, it should not, there is also some figure pdfs and so on. But I kept .pdf out of the gitignore because of the paper pdfs in doc. anyway. I will remove the junk. thanks for pointing this out.